httpretty/__init__.py

```diff
@@ -55,6 +55,14 @@ def last_request():
     """returns the last request"""
     return httpretty.last_request
 
+def all_requests():
+    """Returns list of all requests made"""
+    return httpretty.latest_requests
```

Comment:
On Fri, 11 Dec 2015, Karel Jakubec wrote:
> > +def all_requests():
> > +    """Returns list of all requests made"""
> > +    return httpretty.latest_requests

> Yeah, it would me more consistent with underlying implementation, but what does
> it mean "latest_requests"? How many of them? :) all_requests is simple and
> understandable - just all of them...

I agree.  I can only guess is that 'latest_' was to signal that it might
be not all ever received requests but the requests after 'reset' ;) If
we ever decide to refactor it, I would have called it just 'requests'.
'all_' could be misleading as well in the light of my
'comprehension' above (e.g. it could have literally collected all
responses, disregarding reset) ;)

So for consistency I would have at least kept it inline with httpretty's
internal commands as the rest of its exposed API.